tools/debian/control

```diff
@@ -159,8 +159,7 @@ Depends: ${misc:Depends},
          libpwquality-tools,
          openssl,
 Recommends: policykit-1
-Provides: cockpit-realmd,
```

Comment:
Indeed cockpit-realmd has never been in Debian/Ubuntu as an explicit package, but again it's an unrelated change. Separate commit for cleaning it up is fine if you want to keep it.

Reply:
Nevermind, same reason as for the .spec file, my comment was bogus.

Reply:
Removed this change.

Reply:
Oh, misunderstanding. I put the change back.